packages/react/tsconfig.json

```diff
@@ -16,5 +16,5 @@
     "isolatedModules": true,
     "verbatimModuleSyntax": true
   },
-  "exclude": ["dist/", "examples/*"]
+  "exclude": ["dist/", "examples/*", "*.test.ts"]
```

Comment:
We want to type check tests. Actually, the `dist` and example` here are redundant too.